pkg/scheduler/schedule_one.go

```diff
@@ -812,14 +812,18 @@ func getAttemptsLabel(p *framework.QueuedPodInfo) string {
 // pod has failed to schedule. Also, update the pod condition and nominated node name if set.
 func (sched *Scheduler) handleSchedulingFailure(ctx context.Context, fwk framework.Framework, podInfo *framework.QueuedPodInfo, err error, reason string, nominatingInfo *framework.NominatingInfo) {
 	pod := podInfo.Pod
+	var errMsg string
+	if err != nil {
+		errMsg = err.Error()
+	}
 	if err == ErrNoNodesAvailable {
 		klog.V(2).InfoS("Unable to schedule pod; no nodes are registered to the cluster; waiting", "pod", klog.KObj(pod))
 	} else if fitError, ok := err.(*framework.FitError); ok {
 		// Inject UnschedulablePlugins to PodInfo, which will be used later for moving Pods between queues efficiently.
 		podInfo.UnschedulablePlugins = fitError.Diagnosis.UnschedulablePlugins
-		klog.V(2).InfoS("Unable to schedule pod; no fit; waiting", "pod", klog.KObj(pod), "err", err)
+		klog.V(2).InfoS("Unable to schedule pod; no fit; waiting", "pod", klog.KObj(pod), "err", errMsg)
```

Comment:
I am not sure about this,  `FitError` has other info beside the error msg, I am afraid we will lost it after this change.

Reply:
Fine, `FitError.Error()` will do that for us.

/lgtm